React/wbooks/src/Login/layout.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Field, reduxForm } from 'redux-form';
+
+import './style.css';
+
+const required = value => (value ? undefined : 'Requerido');
+const maxPassLen = 52;
```

Comment:
this kind of constants (numbers, strings, objects imported from a utils file that won't change and don't depend on variables) should be uppercase